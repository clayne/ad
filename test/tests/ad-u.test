ad | -u always | utf8.bin | | 0
