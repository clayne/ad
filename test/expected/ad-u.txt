0000000000000000: E280 9C48 656C 6C6F 2C20 776F 726C 6421  “□□Hello, world!
0000000000000010: E280 9D0A 3132 3334 3536 3738 39E2 989E  ”□□.123456789☞□□
0000000000000020: 0A31 3233 3435 3637 3839 3031 3233 E298  .1234567890123☞□
0000000000000030: 9E0A 3132 3334 3536 3738 3930 3132 33E2  □.1234567890123☞
0000000000000040: 989E 340A E298 31E2 3132 0A              □□4...1.12.
